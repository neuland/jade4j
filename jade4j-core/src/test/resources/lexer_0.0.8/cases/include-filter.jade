html
  body
    include:md some.md
    script
      include:coffee(minify=true) include-filter-coffee.coffee
    script
      include:coffee(minify=false) include-filter-coffee.coffee
