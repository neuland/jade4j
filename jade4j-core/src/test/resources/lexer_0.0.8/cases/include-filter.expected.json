{"type":"tag","line":1,"val":"html","selfClosing":false}
{"type":"indent","line":2,"val":2}
{"type":"tag","line":2,"val":"body","selfClosing":false}
{"type":"indent","line":3,"val":4}
{"type":"include","line":3,"val":"some.md","filter":"md","attrs":null}
{"type":"newline","line":4}
{"type":"tag","line":4,"val":"script","selfClosing":false}
{"type":"indent","line":5,"val":6}
{"type":"include","line":5,"val":"include-filter-coffee.coffee","filter":"coffee","attrs":{"type":"attrs","line":5,"attrs":[{"name":"minify","val":"true","escaped":true}]}}
{"type":"outdent","line":6}
{"type":"tag","line":6,"val":"script","selfClosing":false}
{"type":"indent","line":7,"val":6}
{"type":"include","line":7,"val":"include-filter-coffee.coffee","filter":"coffee","attrs":{"type":"attrs","line":7,"attrs":[{"name":"minify","val":"false","escaped":true}]}}
{"type":"outdent","line":8}
{"type":"outdent","line":8}
{"type":"outdent","line":8}
{"type":"eos","line":8}